### PR TITLE
[13.0][MIG] account_invoice_check_total: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa TT26070

Supersedes https://github.com/OCA/account-invoicing/pull/601

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]
Code and functional review :+1:

**Review by joao-p-marques — APPROVED** [review]

**Comment by joao-p-marques** [comment]
Code and functional review

File: account_invoice_check_total/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account Invoice Check Total",
+    "summary": """
+        Check if the verification total is equal to the bill's total""",
+    "version": "13.0.1.0.0",
+    "website": "https://github.com/OCA/account-invoicing",
+    "author": "Acsone SA/NV, Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline account_invoice_check_total/__manifest__.py:10]:
```suggestion
    "author": "Acsone SA/NV, Tecnativa, Odoo Community Association (OCA)",
```

Comment (pedrobaeza) [reply account_invoice_check_total/__manifest__.py:10]: Not really. The migration itself is not very huge, so Tecnativa should only be contributor in this case.

**Review by victoralmau — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Relaunching after fixing the problem in the branch in f91443693cccb449f8a34607691f51d0fb3e6278:

/ocabot merge nobump

**Pull request merged**